tabs-tabs-tabs/tabs.js

```diff
@@ -70,5 +70,49 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-add-zoom") {
+      callOnActiveTab((tab) => {
+        chrome.tabs.getZoom(tab.id, function(zoomFactor){
+            var newZoomFactor = zoomFactor + 0.2;
+            newZoomFactor = newZoomFactor > 3 ? 3 : newZoomFactor
+
+            //the maximum zoomFactor is 3, it can't go higher
+            if (zoomFactor >= 3) {
+                alert("Tab zoom factor is already at max!");
+            } else {
+                chrome.tabs.setZoom(tab.id, newZoomFactor);
+            }
+        });
+      });
+  }
+
+  else if (e.target.id === "tabs-decrease-zoom") {
+      callOnActiveTab((tab) => {
+        chrome.tabs.getZoom(tab.id, function(zoomFactor){
+            var newZoomFactor = zoomFactor - 0.2;
+            newZoomFactor = newZoomFactor < 0.3 ? 0.3 : newZoomFactor
```

Comment:
You could move these 2 lines inside the "else" at line 98, since you don't need to do this if you are already at max.